recipe/meta.yaml

```diff
@@ -10,25 +10,26 @@ source:
     md5: 5de72a6b8e4cc071ca2154fadf3681e3
 
 build:
-    number: 0
+    number: 1
     script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
     build:
         - python
         - setuptools
-        - h5py
     run:
         - python
-        - h5py
+        # This pins to the default channel version as conda-forge's h5py (hdf5) is incompatible with libnetcdf at the moment.
+        - numpy x.x
+        - h5py 2.6.0 np{{ np }}py{{ py }}_0
```

Comment:
I am not a big fan of this solution. @jakirkham, @pelson, and @msarahan does any of you know a better way to pin h5py to build number `0`? (Ideally we would pin to the default channel.)